src/site/shared/tests/components/ButtonToggle.test.tsx

```diff
@@ -0,0 +1,35 @@
+import "@testing-library/jest-dom";
+import {render}  from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import {ButtonToggle} from "shared/components/ButtonToggle";
+
+import {GetToggles} from "../helpers/GetToggles";
+
+
+describe("Button Toggle", () => {
+    test("Button on", () => {
+        render(<ButtonToggle text="test" isOn />);
+        const [buttonOn, buttonOff] = GetToggles("test");
+        expect(buttonOn).toBeVisible();
+        expect(buttonOff).not.toBeVisible();
+    });
+    test("Button off", () => {
+        render(<ButtonToggle text="test" isOn={false} />);
+        const [buttonOn, buttonOff] = GetToggles("test");
+        expect(buttonOn).not.toBeVisible();
+        expect(buttonOff).toBeVisible();
+    });
+    test("onChange", async () => {
+        let isOn = false;
+        const user = userEvent.setup();
+        render(<ButtonToggle text="test" isOn={isOn} onChange={() => isOn = !isOn} />);
```

Comment:
This test implies that you're changing whether or not the Button is on/off, but setting `isOn` doesn't actually change it to be on/off cause it's not a state variable, so make a different name for the variable